locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocationConfig.java

```diff
@@ -71,13 +72,25 @@
     public static final ConfigKey<String> LOGIN_USER_PRIVATE_KEY_FILE = ConfigKeys.newStringConfigKey("loginUser.privateKeyFile",
             "Custom private key for the user who logs in initially", null); 
     public static final ConfigKey<String> EXTRA_PUBLIC_KEY_DATA_TO_AUTH = ConfigKeys.newStringConfigKey("extraSshPublicKeyData",
-            "Additional public key data to add to authorized_keys", null);
+        "Additional public key data to add to authorized_keys", null);
+    @SuppressWarnings("serial")
+    public static final ConfigKey<List<String>> EXTRA_PUBLIC_KEYS_TO_AUTH = ConfigKeys.newConfigKey(new TypeToken<List<String>>() {}, 
```

Comment:
For consistency with other names, should this be `EXTRA_PUBLIC_KEY_FILES_TO_AUTH`?